crates/rpc/rpc/src/eth/api/server.rs

```diff
@@ -41,7 +41,7 @@ where
 
     /// Handler for: `eth_coinbase`
     async fn author(&self) -> Result<Address> {
-        Err(internal_rpc_err("unimplemented"))
+        Ok(self.pool().status().coinbase)
```

Comment:
Coinbase makes sense to be part of block builder that is still not defined. Execution is building the block but CL is sending the needed parameters that are used to build the block.

Coinbase is send over EngineAPI from CL part. It is part of https://github.com/ethereum/execution-apis/blob/main/src/engine/shanghai.md#payloadattributesv2 :
```
suggestedFeeRecipient: DATA, 20 Bytes - suggested value for the feeRecipient field of the new payload
```
So it can be changed depending on given values from CL. Infur and providers probably disabled it because they are not mining, it is set only if the node is configured as a miner (IIRC).

Reply:
Then, should we close this until block building is defined? Or add it as an attribute of `EthApi` for now, updating it on every payload received from CL?